packages/PHPStanStaticTypeMapper/TypeMapper/ClosureTypeMapper.php

```diff
@@ -43,12 +43,13 @@ public function mapToPHPStanPhpDocTypeNode(Type $type): TypeNode
         return new SpacingAwareCallableTypeNode($identifierTypeNode, [], $returnDocTypeNode);
     }
 
-    /**
-     * @param ClosureType $type
-     */
     public function mapToPhpParserNode(Type $type, ?string $kind = null): ?Node
     {
-        return $this->callableTypeMapper->mapToPhpParserNode($type, $kind);
```

Comment:
with the generics applied, phpstan (and phpstorm) were grumpy about the `CallableTypeMapper` handling a `ClosureType`. it seems in the phpstan type-system a `CallableType` and a `ClosureType` are not compatible.

therefore I killed the dependency on `CallableTypeMapper` and inlined the implementation here instead

Reply:
Cool :+1: 

Btw, I see only removed lines here:

![image](https://user-images.githubusercontent.com/924196/123524370-07f27480-d6ca-11eb-87af-9bcaef161e93.png)

Reply:
hmm ?

![grafik](https://user-images.githubusercontent.com/120441/123524595-bea32480-d6cb-11eb-8058-f65edd5f0299.png)

Reply:
I see, thanks!

Btw, commenting on green line shows the full diff even for me.